src/components/HistoryTables/detail.tsx

```diff
@@ -394,11 +399,13 @@ type SeeMoreContentProps = {
   gasUsed: number
   gasCost: GasCost
   actorName: string
+  executionTrace: ExecutionTrace
 }
 
 SeeMoreContent.propTypes = {
   message: GRAPHQL_MESSAGE_PROPTYPE.isRequired,
   gasUsed: PropTypes.number.isRequired,
   gasCost: GRAPHQL_GAS_COST_PROPTYPE.isRequired,
-  actorName: PropTypes.string.isRequired
+  actorName: PropTypes.string.isRequired,
+  executionTrace: EXECUTION_TRACE_PROPTYPE.isRequired
```

Comment:
You'll notice i added this and pass the prop for now, but it's not being used yet. I'll follow up in another PR with the additional UIs to show subcals